services/vehicle/src/main/java/com/wedriveu/services/vehicle/entity/VehicleStore.java

```diff
@@ -15,14 +15,12 @@
 public interface VehicleStore {
 
     /**
-     * Maps a <em>com.wedriveu.services.vehicle.entity.Vehicle</em> object in a Json object.
-     */
-    void createVehiclesFile();
-
-    /**
+     * A new vehicle is being added to the json file. if the vehicle already exists (the licencePlace is already there)
+     * it sends back a message containing a negative boolean, or positive otherwise.
+     *
      * @param vehicle Indicates the vehicle the will be added to the db.
      */
-    void addVehicle(Vehicle vehicle);
+    void addVehicle(Message vehicle);
```

Comment:
From a design point of view, this is not so good because you're binding a framework (vertx) model (`Message` class) into an interface. Avoid this if is avoidable.